docs/deployment/heroku.rst

```diff
@@ -43,12 +42,23 @@ Updating Currency Exchange Rates
 
 This needs to be run periodically. The best way to achieve this is using Heroku's Scheduler service. Let's add it to our application:
 
-.. code-block:: bash
+.. code-block:: console
 
  $ heroku addons:create scheduler
 
 Then log into your Heroku account, find the Heroku Scheduler addon in the active addon list, and have it run the following command on a daily basis:
 
-.. code-block:: bash
+.. code-block:: console
+
+ $ python manage.py update_exchange_rates --all
```

Comment:
This one is not a console command, it's something you're supposed to copy and paste so I think it's best to not have the `$` and to use a generic `.. code-block::` with no highlighter enabled.